leader-election-impl/src/main/java/com/palantir/paxos/PaxosStateLogBatchReader.java

```diff
@@ -40,7 +39,7 @@ public PaxosStateLogBatchReader(PaxosStateLog<V> delegate, Persistable.Hydrator<
         this.delegate = delegate;
         this.hydrator = hydrator;
         this.executor = MoreExecutors.listeningDecorator(
-                PTExecutors.newFixedThreadPool(numThreads, new NamedThreadFactory("psl-reader", true)));
+                PTExecutors.newFixedThreadPool(numThreads, "psl-reader"));
```

Comment:
Again, no daemons?

Reply:
Yes. this is a change to avoid daemon threads. Do you think non-daemon threads are required here?

Reply:
This one could be either way I think, it's part of the SQLite migration so it blocks startup for timelock, and there'll always be other main threads going around.